### DISCUSSION TITLE
"Multivariate time series classification with sktime" page: deprecated MrSEQClassifier()

### DISCUSSION THREAD

**Discussion by jmwhyte** [alan-turing-institute/sktime#1883] [Q&A]
Hello, I'm new to time series classification and found the [multivariate sktime page](https://github.com/alan-turing-institute/sktime/blob/main/examples/03_classification_multivariate.ipynb) a useful start. 
Having just run it in my freshly upgraded Jupyterlab, and running the final block (under the heading "Bespoke classification algorithms") I see:

FutureWarning: Call to deprecated class MrSEQLClassifier. (MrSEQLClassifier will be removed in v0.10.0. It will be replaced with an implementation based on Numba.) -- Deprecated since version 0.8.1.
  clf = MrSEQLClassifier()

Is there any plan to update the tutorial page to given an option that is likely to have continuing relevance?

**Comment by fkiraly** [comment]
The tutorial has been updated in 0.10.0, and no longer refers to `MrSEQLClassifier`.

Thanks for bringing this up!
